tests/test_sampleset.py

```diff
@@ -241,6 +241,34 @@ def test_from_bqm_single_sample(self):
         self.assertEqual(samples,
                          dimod.SampleSet.from_samples(([-1, 1], 'ab'), dimod.SPIN, energy=1))
 
+    def test_truncate(self):
+        bqm = dimod.BinaryQuadraticModel.from_ising({v: -1 for v in range(100)}, {})
+        samples = dimod.SampleSet.from_samples_bqm(np.triu(np.ones(100)), bqm.binary)
+
+        # by default should be in reverse order
```

Comment:
for this, shouldn't the bqm need to prepped with v:1

Reply:
I do not understand the question. The order stored in the underlying array is in reverse energy order (because of the bqm). So truncating by energy then iterating by energy low-to-high should give low energy solutions (mostly all 1s)

Reply:
I understand now, will fix